### PR TITLE
Variantcalls

### PR DESCRIPTION
**Pull request by shwetagopaul92**
Added and tested getVarinatCalls() which returns the genotype content.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -9,17 +9,34 @@ Authors@R: c(person("Cassie", "Doll", role = c("aut")),
...
+    BSgenome.Hsapiens.UCSC.hg19,
+    org.Hs.eg.db,
+    TxDb.Hsapiens.UCSC.hg19.knownGene
 Description: Provides an R package to interact with the Google Genomics API.
 VignetteBuilder: knitr
 License: Apache License (== 2.0) | file LICENSE
 URL: https://cloud.google.com/genomics/
 BugReports: https://github.com/Bioconductor/GoogleGenomics/issues
-biocViews: DataImport, ThirdPartyClient
+biocViews: DataImport, ThirdPartyClient, genetics
```
Comment (siddharthab) [inline DESCRIPTION]: I don't know if case sensitive matters, but could you change it to "Genetics"?

File: NAMESPACE [hunk 2]
```diff
@@ -1,8 +1,10 @@
 ## Network connection related imports ##
 import(httr)
 importFrom(rjson, fromJSON, toJSON)
+import(methods)
```
Comment (siddharthab) [inline NAMESPACE]: Entire package imports before method imports please, and please maintain alphabetic order. So, it should be

import(httr)
import(methods)
importFrom(rjson, fromJSON, toJSON)
...

import(S4Vectors)
importFrom(Biobase, "sampleNames<-")
importFrom(Biostrings, DNAStringSet, DNAStringSetList)
...

export(getVariantCalls)
export(getVariants)
export(variantsToGRanges)
...

File: .travis.yml [hunk 3]
```diff
@@ -8,4 +8,5 @@ r: bioc-release
 # Secure auth credentials can not be made available for pull requests.
 r_build_args:
 r_check_args: "$(if [[ $TRAVIS_PULL_REQUEST != 'false' ]]; then echo '--no-examples'; fi)"
-
+r_github_packages:
```
Comment (siddharthab) [inline .travis.yml]: Can you please add a comment here like this:
`# survival version >= 2.40.1 is needed by HMisc.`

File: DESCRIPTION [hunk 4]
```diff
@@ -9,17 +9,34 @@ Authors@R: c(person("Cassie", "Doll", role = c("aut")),
 Depends:
     R (>= 3.1.0),
     GenomicAlignments (>= 1.0.1),
-    VariantAnnotation
+    VariantAnnotation,
+    survival (>= 2.40-1)
```
Comment (siddharthab) [inline DESCRIPTION]: I think you can remove survival from this list. Travis will install it because you ask Travis to install the dev version from github.

**Comment by shwetagopaul92** [comment]
@siddharthab I have made the required changes and committed/pushed to the repository. Please let me know if I should issue a pull request again!  Thank you.

**Comment by siddharthab** [comment]
Thank you so much for making the suggested changes. Your contribution is very appreciated.

**Comment by deflaux** [comment]
thank you @shwetagopaul92 !!

**Comment by siddharthab** [comment]
These changes should now be available in bioc-devel repository.

**Comment by shwetagopaul92** [comment]
Thank you!

**Pull request merged**